### DISCUSSION TITLE
Can we Add a Copy-to-Clipboard Button in mjml

### DISCUSSION THREAD

**Discussion by shailesh896** [mjmlio/mjml#2524] [Q&A] [answered]
I want to add "copy link" button to my template, to copy the link to my local clipboard when clicked. Is there any way to achieve this in Mjml?

**Comment by iRyusa** [comment, marked as answer]
You can do that in email as it require some JS.

**Comment by shailesh896** [comment]
@iRyusa , thank you for the reply.
can you explain that bit more. like how? can we call an onClick event?

**Comment by iRyusa** [comment]
Js is blocked in email clients so there’s no way to do so.

**Reply by shailesh896** [reply]
ok. thanks for the confirmation.
